network/state_test.go

```diff
@@ -59,44 +49,3 @@ func (mbs *MockBlockState) LatestHeader() *types.Header {
 		Digest:         [][]byte{{}},
 	}
 }
-
-// MockNetworkState
-type MockNetworkState struct{}
```

Comment:
we should probably keep `MockNetworkState` as well since we will need it soon

Reply:
I have re-added it